package.json

```diff
@@ -5,6 +5,8 @@
   },
   "dependencies": {
     "@dosomething/forge": "~6.6.0",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
```

Comment:
From the issue @deadlybutter linked, it looks like [Elixir hardcodes them](https://github.com/laravel/elixir/blob/aac2fd11fe5b7868dfe23a30d54474bfa5c2f33f/src/Config.js#L332) in the Gulp configuration rather than letting you set them in your `.babelrc`. The transforms should be no-ops if you don't have JSX in your scripts, so no real harm in including.